File: py/Program/Car.py
from Vehicle import Vehicle

class Car(Vehicle):
    __jmlKursi = 0
    __jmlPintu = 0

    def __init__(self, jmlKursi = 0, jmlPintu = 0, plat="", merk="", tahunProduksi=0, warna="", pemilik=""):
        super().__init__(plat, merk, tahunProduksi, warna, pemilik)
        self.__jmlKursi = jmlKursi
        self.__jmlPintu = jmlPintu
    
    def setJmlKursi(self, jmlKursi):
        self.__jmlKursi = jmlKursi
    
    def getJmlKursi(self):
        return self.__jmlKursi
    
    def setJmlPintu(self, jmlPintu):
        self.__jmlPintu = jmlPintu
    
    def getJmlPintu(self):
        return self.__jmlPintu

File: cpp/Program/Car.cpp
#pragma once
#include <bits/stdc++.h>
#include "Vehicle.cpp"

class Car : public Vehicle{
    private:
        int jmlKursi;
        int jmlPintu;
    
    public:
        Car(){
            
        }

        Car(int jmlKursi, int jmlPintu, string plat, string merk, int tahunProduksi, string warna, string pemilik) : Vehicle(plat, merk, tahunProduksi, warna, pemilik){
            this->jmlKursi = jmlKursi;
            this->jmlPintu = jmlPintu;            
        }

        void setJmlKursi(int jmlKursi){
            this->jmlKursi = jmlKursi;
        }

        int getJmlKursi(){
            return this->jmlKursi;
        }

        void setJmlPintu(int jmlPintu){
            this->jmlPintu = jmlPintu;
        }

        int getJmlPintu(){
            return this->jmlPintu;
        }

        ~Car(){
            
        }
};